ddpui/core/dbtautomation_service.py

```
@@ -700,35 +710,66 @@ def sync_sources_for_warehouse_v2(
+                        table,
+                    )
+                    table_output_cols = [
+                        col["name"] for col in wclient.get_table_columns(schema, table)
+                    ]
+                    logger.info(
+                        "Warehouse query [3] end: get_table_columns schema=%s table=%s cols=%s",
+                        schema,
+                        table,
+                        len(table_output_cols),
+                    )
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)